renovate.json

```diff
@@ -1,18 +1,17 @@
 {
   "labels": ["renovate"],
   "extends": ["config:base"],
+  "schedule": ["every weekend"],
```

Comment:
How about scheduling the non-major updates for the weekend and allowing major PRs at any time?  I don't think there's much risk of several of us working on major version bumps at once.
```suggestion
```

Reply:
That is unnecessary because of the dashboard feature. We do not need to get PRs (& have the CI run continuously as long as we are not available). The dashboard will let us create major bumps on-demand. 

Think of it as an invite to an event, and you have to RSVP.